### PR TITLE
creating a governance

### PR DESCRIPTION
**Pull request by miao0miao**
suggesting governance to define the roles in the community, their responsibilities, and how to navigate between them

### CODE REVIEW

File: governance [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# overview
+
+
+Flatcra is a community based project, anyone who wants to prticipate is welcomed. We adopted the (CNCF code of conduct) [https://github.com/cncf/foundation/blob/main/code-of-conduct.md] as we pledge to be an opening and welcoming community for anyone who want to praticipate in it.
```
Comment (tormath1) [inline governance]:
```suggestion
Flatcar is a community based project, anyone who wants to participate is welcomed. We adopted the (CNCF code of conduct) [https://github.com/cncf/foundation/blob/main/code-of-conduct.md] as we pledge to be an opening and welcoming community for anyone who want to participate in it.
```

File: governance [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+### Maintainers
+
+Contribution per-se does not qulified to become a maintainer. Maintainers are first and foremost contributors that have shown they are committed to the long term success of a project. Contributors wanting to become maintainers are expected to be deeply involved in contributing code, pull request review, and triage of issues.
+Maintainership is about building trust with the current maintainers of the project and being a person that they can depend on to make decisions in the best interest of the project in a consistent manner.
+
+Committers wanting to become maintainers are expected to:
+
+- Collaborate well.
+- Demonstrate a deep and comprehensive understanding of Fltcar code base, technical goals, and directions.
+- Actively engage with major Chaos Mesh feature proposals and implementations.
```
Comment (ahrkrak) [inline governance]:
```suggestion
- Actively engage with major Flatcar feature proposals and implementations.
```

File: governance [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+Committers are active community members who are responsible for the overall quality and stewardship of the project. Committers are part of the organization with write access to all repositories. Committers are expected to remain actively involved in the project and participate in voting and discussing of proposed project level changes.
+
+Typically, a potential committer needs to show that they have a sufficient understanding of the project, its objectives, and its strategy. To become a committer, you are expected to:
+- express an intresset to the exsiting maintainers about becoming cecoming a coomiter.
+- Have creted substentials PRs
+
+
+### Maintainers
+
+Contribution per-se does not qulified to become a maintainer. Maintainers are first and foremost contributors that have shown they are committed to the long term success of a project. Contributors wanting to become maintainers are expected to be deeply involved in contributing code, pull request review, and triage of issues.
```
Comment (ahrkrak) [inline governance]:
```suggestion
Contribution per se does not qualify someone to become a maintainer. Maintainers are first and foremost contributors that have shown they are committed to the long term success of a project. Contributors wanting to become maintainers are expected to be deeply involved in contributing code, pull request review, and triage of issues.
```

File: governance [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+Flatcra is a community based project, anyone who wants to prticipate is welcomed. We adopted the (CNCF code of conduct) [https://github.com/cncf/foundation/blob/main/code-of-conduct.md] as we pledge to be an opening and welcoming community for anyone who want to praticipate in it.
+
+
+## Roles and responsibilities
+
+
+### Contributer
+
+
+Contributors are community members of who are engaged. Anyone can contribute to the project and become a contributor, regardless of their skillset. There is no expectation of commitment to the project, no specific skill requirements, and no selection process. There are many ways to contribute to the project, which may be one or more of the following (but not limited to):
```
Comment (ahrkrak) [inline governance]:
```suggestion
Contributors are members of the Flatcar community who are engaged and wish to add to the project. Anyone can contribute to the project and become a contributor, regardless of their skillset. There is no expectation of commitment to the project, no specific skill requirements, and no selection process. There are many ways to contribute to the project, which may be one or more of the following (but not limited to):
```

File: governance (hunk 1 above)
Comment (ahrkrak) [reply governance]:
```suggestion
Flatcar is a community based project, anyone who wants to participate is welcomed. We adopted the (CNCF code of conduct) [https://github.com/cncf/foundation/blob/main/code-of-conduct.md] as we pledge to be an opening and welcoming community for anyone who wants to participate in it.
```

File: governance [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+- Reporting bugs and/or fixing them. 
+- posting feature requests
+- taking an ownership on maintineg spcific packages up to date.
```
Comment (ahrkrak) [inline governance]:
```suggestion
- Taking an ownership on maintineg spcific packages up to date. 
```

File: governance [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+## Roles and responsibilities
+
+
+### Contributer
+
+
+Contributors are community members of who are engaged. Anyone can contribute to the project and become a contributor, regardless of their skillset. There is no expectation of commitment to the project, no specific skill requirements, and no selection process. There are many ways to contribute to the project, which may be one or more of the following (but not limited to):
+
+- Reporting bugs and/or fixing them. 
+- posting feature requests
```
Comment (ahrkrak) [inline governance]:
```suggestion
- Posting feature requests
```

File: governance [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+- taking an ownership on maintineg spcific packages up to date. 
+- updating/writing documentation
```
Comment (ahrkrak) [inline governance]:
```suggestion
- Updating/writing documentation
```

File: governance [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+- submitting pull requests or patches
```
Comment (ahrkrak) [inline governance]:
```suggestion
- Submitting pull requests or patches
```

File: governance [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+- Improving the Flatcar website.
+- Joining discussions on Twitter, our community [Slack channel](https://kubernetes.slack.com/archives/C03GQ8B5XNJ) ,[Matrix channel](https://app.element.io/#/room/#flatcar:matrix.org) and office hours. 
+- Evangelizing about the project. 
+
+For first-time contributors, it’s recommended to start by going through [Contribuiting-git](https://github.com/flatcar-linux/Flatcar/blob/main/contributions-git.md), and joining our community Slack channel or Matrix channel.
```
Comment (ahrkrak) [inline governance]:
```suggestion
For first-time contributors, it’s recommended to start by going through [Contributing-git](https://github.com/flatcar-linux/Flatcar/blob/main/contributions-git.md), and joining our community Slack channel or Matrix channel. 
```

File: governance [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+For first-time contributors, it’s recommended to start by going through [Contribuiting-git](https://github.com/flatcar-linux/Flatcar/blob/main/contributions-git.md), and joining our community Slack channel or Matrix channel. 
+
+
+### Committers
+
+Committers are active community members who are responsible for the overall quality and stewardship of the project. Committers are part of the organization with write access to all repositories. Committers are expected to remain actively involved in the project and participate in voting and discussing of proposed project level changes.
+
+Typically, a potential committer needs to show that they have a sufficient understanding of the project, its objectives, and its strategy. To become a committer, you are expected to:
+- express an intresset to the exsiting maintainers about becoming cecoming a coomiter.
```
Comment (ahrkrak) [inline governance]:
```suggestion
- express an interest to the existing maintainers about becoming becoming a committer; and
```

File: governance [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+- Have creted substentials PRs
```
Comment (ahrkrak) [inline governance]:
```suggestion
- have created substantial PRs.
```

File: governance [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+
+### Maintainers
+
+Contribution per-se does not qulified to become a maintainer. Maintainers are first and foremost contributors that have shown they are committed to the long term success of a project. Contributors wanting to become maintainers are expected to be deeply involved in contributing code, pull request review, and triage of issues.
+Maintainership is about building trust with the current maintainers of the project and being a person that they can depend on to make decisions in the best interest of the project in a consistent manner.
+
+Committers wanting to become maintainers are expected to:
+
+- Collaborate well.
+- Demonstrate a deep and comprehensive understanding of Fltcar code base, technical goals, and directions.
```
Comment (ahrkrak) [inline governance]:
```suggestion
- Demonstrate a deep and comprehensive understanding of the Flatcar code base, technical goals, and directions.
```

File: governance [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+- Actively engage with major Chaos Mesh feature proposals and implementations.
+
+Periodically, the existing maintainers curate a list of contributors that have shown regular activity on the project over the prior months. The nominating maintainer will create a PR to update the Maintainers List. It is recommended to describe the reasons for the nomination and the contribution of the nominee in the PR. Upon consensus of incumbent maintainers, the PR will be approved and the new maintainer becomes active.
```
Comment (ahrkrak) [inline governance]: delete this para? I don't think we do this.

**Review by ahrkrak — CHANGES_REQUESTED** [review]

File: governance (hunk 13 above)
Comment (miao0miao) [reply governance]: I want to also lay the foundation for growth and establish a clear path for becoming a maintainer.

Comment (miao0miao) [reply governance]: @ahrkrak I think we should keep it, any objection?

File: governance.md [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+- Reporting bugs and/or fixing them. 
+- Posting feature requests
+- Taking an ownership on maintineg spcific packages up to date. 
+- Updating/writing documentation
+- Submitting pull requests or patches
+- Improving the Flatcar website.
+- Joining discussions on Twitter, our community [Slack channel](https://kubernetes.slack.com/archives/C03GQ8B5XNJ) ,[Matrix channel](https://app.element.io/#/room/#flatcar:matrix.org) and office hours. 
+- Evangelizing about the project. 
+
+For first-time contributors, it’s recommended to start by going through [Contributing-git](https://github.com/flatcar-linux/Flatcar/blob/main/contributions-git.md), and joining our community Slack channel or Matrix channel.
```
Comment (ahrkrak) [inline governance.md]:
```suggestion
For first-time contributors, it’s recommended to start by going through [Contributing-git](https://github.com/flatcar/Flatcar/blob/main/contributions-git.md), and joining our community Slack channel or Matrix channel. 
```

**Review by ahrkrak — CHANGES_REQUESTED** [review]

**Comment by ahrkrak** [comment]
lgtm

**Pull request merged**